app/src/main/java/com/guerinet/mymartlet/util/dbflow/DBUtils.java

```diff
@@ -49,7 +50,7 @@
      * @param <T>        Object type
      */
     public static <T extends BaseModel> void replaceDB(Context context, String dbName,
-            Class<T> type, List<T> newObjects, @Nullable Callback callback) {
+                                                       Class<T> type, List<T> newObjects, @Nullable Callback callback) {
```

Comment:
You should be able to. I'm actually looking into that right now for the new Android developer at my job so I'll keep you updated.